evm/contracts-deployment-test.js

```diff
@@ -0,0 +1,60 @@
+'use strict';
+
+const { ethers } = require('hardhat');
+const { execSync } = require('child_process');
+const {
+    Wallet,
+    providers: { JsonRpcProvider },
+} = ethers;
+const { Command, Option } = require('commander');
+
+const { mainProcessor } = require('./utils');
+const { addExtendedOptions } = require('./cli-utils');
+
+async function processCommand(_, chain, options) {
+    const wallet = new Wallet(options.privateKey, new JsonRpcProvider(chain.rpc));
+
+    const cmds = [
+        `node evm/deploy-contract.js -c ConstAddressDeployer -m create --artifactPath ../evm/legacy/ConstAddressDeployer.json`,
+        `node evm/deploy-contract.js -c Create3Deployer -m create2`,
+        `node evm/deploy-gateway-v6.2.x.js -m create3 --keyID ${wallet.address}`,
+        `node evm/gateway.js --action params`,
+        `node evm/deploy-contract.js -c Operators -m create2`,
+        `node evm/deploy-upgradable.js -c AxelarGasService -m create${options.env === 'testnet' ? '' : '2'}`,
+        `node evm/deploy-contract.js -c Multisig -m create3 -s 'testSalt'`,
+        `node evm/deploy-contract.js -c InterchainGovernance -m create3`,
+        `node evm/deploy-its.js -s "testSalt" --proxySalt 'testSalt'`,
+    ];
```

Comment:
Why not also send the transactions to transfer ownership of Multisig and InterchainGovernance to the gateway?